cmd/package-operator-manager/components/components_test.go

```diff
@@ -34,3 +34,24 @@ func TestUncachedClient(t *testing.T) {
 	require.EqualError(t, err,
 		"unable to set up uncached client: must provide non-nil rest.Config to client.New")
 }
+
+func TestProvideRestConfig(t *testing.T) {
+	t.Parallel()
+
+	_, err := ProvideRestConfig()
+	require.Error(t, err)
```

Comment:
What error are we expecting here?

Reply:
These errors are left over the way other tests in file has been left.